src/cascadia/TerminalApp/TabRowControl.h

```diff
@@ -25,7 +24,5 @@ namespace winrt::TerminalApp::implementation
 
 namespace winrt::TerminalApp::factory_implementation
 {
-    struct TabRowControl : TabRowControlT<TabRowControl, implementation::TabRowControl>
-    {
-    };
+    BASIC_FACTORY(TabRowControl);
```

Comment:
Outside the scope of this... but maybe we should prefix these macros with something (like how the WIL ones do WI_FLAG_SET) so we understand which ones are a part of our magic and which ones are a part of the actual cppwinrt packages.

Reply:
> so we understand which ones are a part of our magic and which ones are a part of the actual cppwinrt packages.

> implying anything useful comes with cppwinrt